lib/script-loader.php

```diff
@@ -79,7 +82,7 @@ function gutenberg_enqueue_global_styles() {
  */
 function gutenberg_enqueue_global_styles_custom_css() {
 	_deprecated_function( __FUNCTION__, 'Gutenberg 17.8.0', 'gutenberg_enqueue_global_styles' );
-	if ( ! wp_is_block_theme() ) {
+	if ( ! wp_is_block_theme() || is_customize_preview() ) {
```

Comment:
As noted in https://github.com/WordPress/wordpress-develop/pull/9000#discussion_r2169925725:

Actually, I think this change can be undone. It's a deprecated function anyway so it is not used. And the logic here is not right as it should rather be doing something like this:

```php
$custom_css = '';
if ( ! is_customize_preview() ) {
	// Don't enqueue Customizer's custom CSS separately.
	remove_action( 'wp_head', 'wp_custom_css_cb', 101 );

	$custom_css .= wp_get_custom_css();
}

$custom_css .= wp_get_global_styles_custom_css();
```

But again, given that this function is deprecated it doesn't seem necessary to make this change.

```suggestion
	if ( ! wp_is_block_theme() ) {
```